.github/workflows/ci.yml

```diff
@@ -110,7 +112,7 @@ jobs:
         with: { go-version: "${{ matrix.language_version }}" }
 
       - if: matrix.language == 'rust'
-        run: rustup default ${{ matrix.language_version }} && rustup component add clippy rustfmt
+        run: rustup default ${{ matrix.language_version }} && rustup component add clippy rustfmt && cargo +stable install cargo-audit
```

Comment:
Ideally, we'd get the number of deps to zero, instead of auditing them. Perhaps we can _just_ drop `cargo-audit`? It's most valuable if you have a bunch of dependencies from random maintainers, but we only have a few, and it's unlikely that they'll be compromised? In any case, we are shipping a library, so we don't actually have much control over deps, it's the consumers that determine versions.